lib/Parser/ptree.h

```diff
@@ -433,7 +433,7 @@ enum FncFlags : uint
     kFunctionIsClassConstructor                 = 1 << 18, // function is a class constructor
     kFunctionIsBaseClassConstructor             = 1 << 19, // function is a base class constructor
     kFunctionIsClassMember                      = 1 << 20, // function is a class member
-    // Free = 1 << 21,
+    kFunctionHasCachedScope                     = 1 << 21, // Function has cached scopes
```

Comment:
Can you point out the benefit of having this recorded on the parse node, since the byte code generator is already recording the attribute on the FuncInfo?

Reply:
Used for creating the attributes in GetFunctionInfoAttributes